src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,7 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList extends ArrayList<Task> {
+    private static final long serialVersionUID = 1L;
```

Comment:
[Because exception implements Serializable, the serialVersionUID should be declared as it is.](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html)

Reply:
Oh I see! Sorry about that I did not know about the Serializable interface.